service/src/main/java/uk/gov/hmcts/reform/fpl/service/docmosis/NoticeOfHearingGenerationService.java

```diff
@@ -51,4 +54,12 @@ private String getHearingType(HearingBooking hearingBooking) {
             hearingBooking.getTypeDetails();
     }
 
+    //feature toggle
+    private String getHearingNotes(String multiHearingNotes, String singleHearingNotes) {
+        if (featureToggleService.isMultiPageHearingEnabled()) {
+            return defaultIfNull(multiHearingNotes, "");
```

Comment:
The isBlank in the document template should be able to handle nulls

Reply:
will check that